README.md

```diff
@@ -108,6 +108,13 @@ $ docker run --env-file ./.env -d --publish 5000:5000 --mount type=bind,source="
 
 $ docker build --target production --tag do-me:prod .
 $ docker run --env-file ./.env -d --publish 5000:5000 do-me:prod
+
+$ docker build --target test --tag do-me:test .
+$ docker run --env-file ./.env -it do-me:test todo_app/tests
```

Comment:
Minor but I think this should be the test version of your environment files, since you rely on the board name being the same as that file
```suggestion
$ docker run --env-file ./.env.test -it do-me:test todo_app/tests
```
Would be nice to also add the e2e test command explicitly too,
```
docker run --env-file ./.env -it do-me:test todo_app/tests_e2e
```